docs/pages/guides/dom-components.mdx

```diff
@@ -192,7 +196,35 @@ export default function App() {
 
 ## Measuring DOM components
 
-You may want to measure the size of a DOM component and report it back to the native side (for example, native scrolling). This can be done using a native action:
+You may want to measure the size of a DOM component and report it back to the native side (for example, native scrolling). This can be done using a `withDOMAutoSize` HOC or a manual native action:
```

Comment:
```suggestion
You may want to measure the size of a DOM component and report it back to the native side (for example, native scrolling). This can be done using a `withDOMAutoSize` higher-order component (HOC) or a manual native action:
```